bazel/markdown_to_html/src/extensions/docs-code/sanitizers/eslint.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+export type FileType =
+  | 'ts'
+  | 'js'
+  | 'mjs'
+  | 'es6'
+  | 'html'
+  | 'svg'
+  | 'css'
+  | 'conf'
+  | 'yaml'
+  | 'yml'
+  | 'sh'
+  | 'jade'
+  | 'pug'
+  | 'json'
+  | 'json.annotated';
+
+export function removeEslintComments(input: string, fileType: FileType) {
```

Comment:
It would be good to include a comment on why this is necessary

(I'm assuming we run eslint on the code examples, and suppress them for.... reasons? )